content/chat/rooms/index.textile

```diff
@@ -11,31 +11,38 @@ languages:
 
 Rooms are used to organize and logically separate your users and chat messages into 'rooms'. They are the entry object into using chat and provide access to all other chat features, such as messages, online status and typing indicators. A room can represent a 1:1 chat between an agent and a customer, a private message between two users in a chat application, a large group conversation, or the chat section of a livestream with thousands of users.
 
-You also control which features are enabled in a room when you create one. For instance, you might only want to utilize typing indicators in direct messages between users.
+h2(#channel-relationship). Channel <> Room relationship
+
+It is important to note that each room is backed by a single Ably Pub/Sub channel.
+The channel name is the same as the room ID with an appended suffix of @::$chat@ (e.g @some-room-id::$chat@). In most cases, you will not need to worry about this, as the Chat SDK handles the channel creation and management for you and capabilities can be configured at the room level.
+
+<aside data-type='note'>
+<p>You may need to be aware of this when using some integrations with Ably, such as the "Pulsar":https://ably.com/docs/integrations/streaming/pulsar#creating-pulsar-rule connectors, or if you are operating in an environment where a Chat SDK is not available.</p>
+</aside>
 
 h2(#create). Create or retrieve a room
 
-Users send messages to a room and subscribe to the room in order to receive messages. Other features, such as indicating which users are online, or which users are typing are configured as part of a room's options.
+Users send messages to a room and subscribe to the room in order to receive messages. Other features, such as indicating which users are online, or which users are typing can be configured as part of a room's options.
```

Comment:
I'm not sure the second sentence is correct here. Isn't typing enabled by default now?